prac_06/guitar.py

```diff
@@ -4,20 +4,18 @@
 class Guitar:
     """Construct a guitar instance"""
 
-    def __init__(self, name="", year=0, cost=0):
+    def __init__(self, name="", year=0, cost=0.0):
```

Comment:
Do we need to initialize as 0.0?  
![image](https://github.com/Lucas-obrien/cp1404practicals/assets/39152811/f0df4e93-85c1-414a-9984-741654fe3047)

Reply:
I was thinking there would be issues with changing data types in the class, but apparently not!
Will change!